### PR TITLE
Added inline_connection_subtype (update locationlog)

### PR DESCRIPTION
**Pull request by fdurand**
# Description

Added Inline sub connection type in locationlog
# Impacts

Locationlog 
# Delete branch after merge

YES
# NEWS file entries
## Enhancements
- Added inline in sub_connection_type (locationlog)

### CODE REVIEW

File: lib/pf/constants/eap_type.pm [hunk 1]
```diff
@@ -84,7 +84,8 @@ Readonly::Scalar our $EAP_AKA2 => 50;
 Readonly::Scalar our $EAP_GPSK => 51;
 Readonly::Scalar our $EAP_PWD => 52;
 Readonly::Scalar our $EAP_EVEV1 => 53;
-
+Readonly::Scalar our $INLINE_L2 => 54;
```
Comment (dwlfrth) [inline lib/pf/constants/eap_type.pm]: Purpose ?

File: lib/pf/constants/eap_type.pm [hunk 2]
```diff
@@ -144,6 +145,8 @@ our %RADIUS_EAP_TYPE_2_VALUES = (
     "Microsoft-MS-CHAPv2" => 26,
     "Cisco-MS-CHAPv2" => 29,
     "MS-CHAP-V2" => 26,
+    "inlinel2" => 54,
```
Comment (dwlfrth) [inline lib/pf/constants/eap_type.pm]: Purpose ?

**Comment by dwlfrth** [comment]
Looks good outside of the 'lib/pf/constants/eap_type.pm'

File: lib/pf/constants/eap_type.pm (hunk 2 above)
Comment (fdurand) [inline lib/pf/constants/eap_type.pm]: Need it for sub_connection_type, but need to be changed to 257

File: lib/pf/constants/eap_type.pm (hunk 1 above)
Comment (fdurand) [inline lib/pf/constants/eap_type.pm]: Need it for sub_connection_type, but need to be changed to 257

**Comment by dwlfrth** [comment]
One thing that is "bugging me" is the presence of inline constant numbers in a "eap_type" library

**Comment by julsemaan** [comment]
What bugs me is that the constants don't seem to be used

**Comment by dwlfrth** [comment]
That's the whoooooole point

**Pull request merged**